octo/builder_dependencies.go

```diff
@@ -94,14 +94,11 @@ func contributeBuildImage(descriptor Descriptor, image string, classifier string
 						Run:  StatikString("/install-update-build-image-dependency.sh"),
 					},
 					{
-						Name: "Install crane",
-						Run:  StatikString("/install-crane.sh"),
-						Env:  map[string]string{"CRANE_VERSION": CraneVersion},
-					},
-					{
-						Name: "Install yj",
-						Run:  StatikString("/install-yj.sh"),
-						Env:  map[string]string{"YJ_VERSION": YJVersion},
+						Uses: fmt.Sprintf("buildpacks/github-actions/setup-tools@v%s", BuildpackActionsVersion),
```

Comment:
nice, is that already published?

Reply:
Yes, it's upstream. Its the same code. The only downfall is that it's bundled together so you can't get just pack or just yj. In some cases we need both, but not always. It seems like a very small price to pay though as the install is very quick (a single curl).

Reply:
yeah, I used to think about using `pack` own github action, but... you're right, it's just curl https://github.com/buildpacks/github-actions?tab=readme-ov-file#setup-pack-cli-action